site/docs/encoding/scale.md

```diff
@@ -321,6 +321,38 @@ For example, the following plot has a binned field on the `color` channel.
+The resulting quantile boundaries segment the domain into groups with roughly equals numbers of sample points per group. The `range` property, if not specified, is default to 4 uniform segments between minimum and maximum range value determined by Vega-lite based on mark types.
+
+Quantile scales are particularly useful for creating color or size encodings with a fixed number of output values. Using a discrete set of encoding levels (typically between 5-9 colors or sizes) sometimes supports more accurate perceptual comparison than a continuous range.
+For related functionality see [quantize scales](https://vega.github.io/vega-lite/docs/scale.html#quantize), which partition the domain into uniform domain extents, rather than groups with equal element counts.
+Quantile scales have the benefit of evenly distributing data points to encoded values. In contrast, quantize scales uniformly segment the input domain and provide no guarantee on how data points will be distributed among the output visual values.
+
+<span class="vl-example" data-name="bar_scale_quantile"></span>
+
+{:#quantize}
+### Quantize Scales
+
+Quantize scales (`"quantize"`) are similar to [linear scales](https://vega.github.io/vega-lite/docs/scale.html#linear), except they use a discrete rather than continuous range. The continuous input domain is divided into uniform segments based on the number of values in (i.e., the cardinality of) the output range.
+Each range value y can be expressed as a quantized linear function of the domain value x: _y_ = _m round(x) + b_. The `range` property, if not specified, is default to 4 uniform segments between minimum and maximum range value determined by Vega-lite based on mark types.
+
+Quantize scales are particularly useful for creating color or size encodings with a fixed number of output values. Using a discrete set of encoding levels (typically between 5-9 colors or sizes) sometimes supports more accurate perceptual comparison than a continuous range.
+For related functionality see [quantile scales](https://vega.github.io/vega-lite/docs/scale.html#quantile), which partition the domain into groups with equal element counts, rather than uniform domain extents.
+
+<span class="vl-example" data-name="bar_scale_quantize"></span>
+
+{:#threshold}
+### Threshold Scales
+
+Threshold scales (`"threshold"`) are similar to [quantize scales](https://vega.github.io/vega-lite/docs/scale.html#quantize), except they allow mapping of arbitrary subsets of the domain (not uniform segments) to discrete values in the range.
+The input domain is still continuous, and divided into slices based on a set of threshold values provided to the domain property. The `range` property must have N+1 elements, where N is the number of threshold boundaries provided in the domain. If the `range` property is not specified, is default to 4 uniform segments between minimum and maximum range value determined by Vega-lite based on mark types.
```

Comment:
> is default to 4 uniform segments between minimum and maximum range value 

It's not 4 anymore right?  Looks like you do N+1 


> determined by Vega-lite based on mark types.

Reading this is still a bit confusing how is it based on mark type.  Perhaps, just say that by default Vega-Lite provides N+1 values that provide N uniform segment as the range?

Reply:
Done.